.github/workflows/build-ci.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
         REDIS_IMAGE=ghcr.io/${{ github.repository }}/redis-ci:${{ github.sha }}
 
   vault-ci:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
why not v24? (curious if there is a specific reason)

Reply:
Playing it safe, I know that there's a bunch of changes in ubuntu `24.04` and due to the situation we are right now, I rather we update it in two steps.